shared/templates/file_groupowner/ansible.template

```diff
@@ -5,32 +5,41 @@
 # disruption = low
 
 {{% for path in FILEPATH %}}
-{{% if IS_DIRECTORY and FILE_REGEX %}}
+{{% if IS_DIRECTORY %}}
+{{% if FILE_REGEX %}}
+
+- name: Find {{{ path }}} file(s) matching {{{ FILE_REGEX[loop.index0] }}}{{% if RECURSIVE %}} recursively{{% endif %}}
```

Comment:
Should this also get symlinks? My interruption of the [docs](https://docs.ansible.com/ansible/2.9/modules/find_module.html#parameter-file_type) says that this will not get symlinks.

Reply:
Hmm, I'm no sure, but my guess is no.
The state of symlink handkling is a mess and confusing to me.

The OVAL doesn't report symlinks as objects, but I don't know if it will follow them.
The Bash remediation doesn't follow symlinks by default, but we make it follow with `-L`, and we make changes to the symlinks when they are found. (but OVAL doesn't care about them)
The Ansible remediation doesn't follow symlinks by default and we don't make them follow.

And looking at the rule description I'm not sure what is the expectation.
If there is a link from `/usr/lib/` to somewhere else, should we go there, check and remediate stuff?

Reply:
I think there  a lot of question to answer regarding symlinks:
- Should we check and report the files/dirs they point to?
- Should we check the symlinks themselves?
- Should we follow them?

Besides https://github.com/ComplianceAsCode/content/pull/8404/commits/c715e1239885932960a4848d7bc2152aa0b0eed5, this PR doesn't do much about the symlink situation.
So the situation will stay almost the same... I think this subject should be revisited in a future PR.

Reply:
Sounds good. Agree we can figure out the symlink issue later.